### DISCUSSION TITLE
Are previously merged pull requests being removed?

### DISCUSSION THREAD

**Discussion by ltguillaume** [rustdesk/rustdesk#8824] [Q&A] [answered]
1. The latest releas 1.2.7 states:
>Shared image of clipboard https://github.com/rustdesk/rustdesk/pull/8561

Looking up this pull request results in a 404 message.

2. Similarly:
>Set clipboard text of controlled Android ([#8677](https://github.com/rustdesk/rustdesk/pull/8677))
>Android actions to menu and with more actions in case floating toolbar is hidden ([#8512](https://github.com/rustdesk/rustdesk/pull/8512))

404

4. The discussion https://github.com/rustdesk/rustdesk/discussions/6444#discussioncomment-9033485 states:
Thanks, https://github.com/rustdesk/rustdesk/pull/7642

This also results in a 404.

What is going on? Is this a GitHub problem? Are you requesting pull request removals?

**Comment by rustdesk** [comment, marked as answer]
>  Lin told me by email thst Github has flagged his account

https://github.com/rustdesk/rustdesk/pull/8764#issuecomment-2240995908

**Reply by ltguillaume** [reply]
And did they get a reason for being flagged?
